src/cmds/service/service.c

```diff
@@ -192,44 +241,36 @@ int main(int argc, char **argv) {
 
 	command = argv[arg_offset];
 
-    /* Stop command */
-    if (arg_offset+1 < argc)
-        if (strcmp(argv[arg_offset+1], "stop") == 0) {
-            service_node = service_lookup_by_name(command);
-            if (service_node){
-                /* Is the task still up or it has exited since last start */
-                if (!is_service_up(service_node)) {
-                    dlist_del_init(&service_node->dlist_item);
-                    pool_free(&pool, service_node);
-                    printf("Service has exited\n");
-                } else { /* If up then stop */
-                    service_delete(service_node);
-                }
-            } else {
-                printf("Cannot stop %s, no such service running\n", command);
-            }
-            return 0;
+    if (arg_offset+1 < argc){
+        service_argv=malloc((argc-arg_offset)*sizeof(void*));
+        /* Prepare arguments for service */
+        for (int i=0; i<argc-arg_offset-1; i++){
```

Comment:
`argc - arg_offset` is repeated a lot of times, I think code would be more readable if it will be a variable with some sensible name